src/shared/SearchBar/component/SearchForm.scoped.css

```diff
@@ -0,0 +1,72 @@
+*, *::before, *::after {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;    
+}
+
+.container-searchForm { 
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    top: 5px;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: #fdffff;
+    border-radius: 10px;
+    width: 924px;
+    box-shadow: 0 1px 3px 0 rgba(0,0,0,.08),0 1px 3px 0 rgba(0,0,0,.08);
+}
+
+.d-flex {
+    justify-content: center;
+    height: 44px;
+    align-items: center;
+    border-bottom: 1px solid #DDDDDD;
+    width: 100%;
+}
+
+.icon {
+    cursor: pointer;
+}
+
+header {
+    padding: 16px;
+    width: 924px;
+}
+input {
+    width: 800px;
+    height: 22px;
+    margin-left: 20px;
+    margin-right: 20px;
+    border: none;
+    font-size: 1rem;
+    color: 595859;
+}
+
+input:focus {
+    outline: none;
+}
+
+ul {
+    line-height: 2.5rem;
+}
+
+li {
+    list-style: none;
+    padding-left: 30px;
+}
+
+li:hover {
+    background-color: #1164A3;
+    color:  #fdffff;
+}
+

```

Comment:
paki add to na css

`li:last-child:hover {
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}`